packages/ui/components/ui/tx/view/viewbox.tsx

```diff
@@ -9,6 +9,8 @@ export interface ViewBoxProps {
   visibleContent?: React.ReactElement;
 }
 
+const Label = ({ label }: { label: string }) => <span className='text-lg'>{label}</span>;
```

Comment:
Just making the label a little bigger, to differentiate from the view box's contents